src/Sticky/Sticky.doc.js

```diff
@@ -19,7 +19,7 @@ card(
       },
       {
         name: 'children',
```

Comment:
This should probably be `children?` if we're going to change it.

Reply:
children shouldn't be optional for sticky imo